docs/Reference/Principles.md

```diff
@@ -16,63 +16,58 @@ the following technical principles:
+as these usually incur some overhead.
 
 ## "Good" Developer Experience
 
-Fastify aims to provide the best developer experience at the performance point
-it is operating.
-It provides a great out-of-the-box experience that is flexible enough to be
-adapted to a variety of situations.
+Fastify aims to provide the best developer experience at its performance point.
+It offers a great out-of-the-box experience that is flexible enough to adapt to
+various situations.
 
-As an example, this means that binary addons are forbidden because most JavaScript
-developers would not
-have access to a compiler.
+For example, binary addons are forbidden because most JavaScript developers do
+not have access to a compiler.
 
 ## Works great for small and big projects alike
 
-We recognize that most applications start small and become more complex over time.
-Fastify aims to grow with
-the complexity of your application, providing advanced features to structure
-your codebase.
+Most applications start small and become more complex over time. Fastify aims to
+grow with this complexity, providing advanced features to structure codebases.
 
 ## Easy to migrate to microservices (or even serverless) and back
 
-How you deploy your routes should not matter. The framework should "just work".
+Route deployment should not matter. The framework should "just work".
 
 ## Security and Data Validation
 
-Your web framework is the first point of contact with untrusted data, and it
-needs to act as the first line of defense for your system.
+A web framework is the first point of contact with untrusted data and must act
+as the first line of defense for the system.
 
 ## If something could be a plugin, it likely should
 
-We recognize that there are an infinite amount of use cases for an HTTP framework
-for Node.js. Catering to them in a single module would make the codebase unmaintainable.
-Therefore we provide hooks and options to allow you to customize the framework
-as you please.
+Recognizing the infinite use cases for an HTTP framework, catering to all in a
+single module would make the codebase unmaintainable. Therefore, hooks and
+options are provided to customize the framework as needed.
 
 ## Easily testable
 
 Testing Fastify applications should be a first-class concern.
 
 ## Do not monkeypatch core
 
-Monkeypatch Node.js APIs or installing globals that alter the behavior of the
-runtime makes building modular applications harder, and limit the use cases of Fastify.
-Other frameworks do this and we do not.
+Monkeypatching Node.js APIs or installing globals that alter the runtime makes
+building modular applications harder and limits Fastify's use cases. Other
+frameworks do this; Fastify does not.
 
 ## Semantic Versioning and Long Term Support
 
-We provide a clear Long Term Support strategy so developers can know when to upgrade.
+A clear Long Term Support strategy is provided to inform developers when
```

Comment:
We should link to the appropriate document here.

Reply:
Added in https://github.com/fastify/fastify/pull/5951/commits/667562a5009cccc7148fa2df03560a445a2a44d6